userland/examples/tests/max17205/main.c

```diff
@@ -0,0 +1,60 @@
+#include <math.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <console.h>
+#include <max17205.h>
+#include <tock.h>
+
+int main (void) {
+  printf("[MAX17205] Test App\n");
+
+  uint8_t rom_id[8];
```

Comment:
Seems safer for this to be a `uint64_t` and just pass a pointer to that. But this is all in the wash if we can just get rid of the allow instead.

Reply:
yes it's certainly better to use a uint64_t. To be honest I didn't know that I could use a 64 bit type.